### DISCUSSION TITLE
unable to run script with LSF executor - error for project

### DISCUSSION THREAD

**Discussion by complexgenome** [nextflow-io/nextflow#4135] [Q&A] [answered]
Hi,
I'm new to nextflow and first time setting up the config file for an executor. I'm on a LSF system. When I request an interactive node I use following command:

`bsub -L /bin/bash -Ip -n 2 -P acc_pareks02a -q premium -R rusage[mem=80GB] -R span[hosts=1] -W 56:00 /bin/bash`

That is: two CPUs, project/lab - pareks02a queue premium, memory, node using hosts=1 and walltime. 
In the `nextflow.config` I provide following:

```
profiles {
	minerva {
		process.executor = 'LSF'
		process.queue = 'premium'
        process.memory = '50GB'
		process.cpus = 2
		process.time = '40 h'
		clusterOptions = { " -P acc_pareks02a "}
	}
}
```
I use clusterOptions to mention the project for internal settings of the cluster. 

I run as:  `nextflow run main.nf -profile minerva`
It errors out as:

```
Caused by:
  Failed to submit process to grid scheduler for execution
Command executed:
  bsub
Command exit status:
  255
Command output:
 Project name must be specified using bsub -P.
```

If I look at `.command.run` file I see:

> #!/bin/bash
> #BSUB -o /sc/arion/projects/pareks02a/sanjeev/nextflow_pipeline/scripts_pipeline/test_pipeline_screen/work/8f/71f5163cd7d8c8c19eee9545c40492/.command.log
> #BSUB -q premium
> #BSUB -n 2
> #BSUB -R "span[hosts=1]"
> #BSUB -W 40:00
> #BSUB -M 25600
> #BSUB -R "select[mem>=51200] rusage[mem=51200]"
> #BSUB -J nf-FASTP_(1)
>  NEXTFLOW TASK: FASTP (1)
> set -e
> set -u
> NXF_DEBUG=${NXF_DEBUG:=0}; [[ $NXF_DEBUG > 1 ]] && set -x
> NXF_ENTRY=${1:-nxf_main}
> 
 
How do I proceed or provide necessary flag information in the nextflow.config file to incorporate `-P` flag?

**Comment by bentsherman** [comment]
Use the `clusterOptions` directive

**Comment by complexgenome** [comment]
Hi @bentsherman Thanks for your reply. 
In the code above, I used `clusterOptions` I've tried removing `{}` but didn't work, also used single quote but no success.

**Reply by bentsherman** [reply, marked as answer]
It is a process directive so should be `process.clusterOptions`. Also don't think you need the curly braces here since there is no variable interpolation

**Comment by complexgenome** [comment]
@bentsherman  Great, it works now.  Thank you 🙏 
One last question - Would each process be submitted as a job? For instance, process1 is submitted successfully . Process2 takes input as output of process1. Would process2 be also submitted as an independent job?

**Reply by bentsherman** [reply]
Yes exactly
